python/lib/sift_py/data_import/csv.py

```diff
@@ -170,10 +203,20 @@ def simple_upload(
                 raise Exception(
                     f"Unable to upload data type in column {i+1} {header}. Inferred type: {inferred_dtype}"
                 )
+            if dtype == "int":
```

Comment:
What's your thinking for using `infer_dtype()` and doing this vs using `df.dtypes` for getting the datatype for each column?

Reply:
No real reason, this was just the first way I tried.

Reply:
Gotcha! Would you be willing to try using `df.dtypes` to see if that would simplify things here? I wonder if we wouldn't need the `is_uint64` check and could save a few lines there.